force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +250,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
-
+        List<Integer> numList = new List<Integer>();
         // Add the integers 1, 2, and 3 to the list
         // add 1
+        numList.add(1);
         // add 2
+        numList.add(2);
         // add 3
+        numList.add(3);
```

Comment:
Try adding the numbers in the constructor instead of after the list is created https://salesforce.stackexchange.com/questions/203694/how-to-initialize-list-set-values-by-construction-in-apex